### ISSUE TITLE
No Enter/Exit Live Reveal Slideshow

### ISSUE THREAD

**Issue by lbugnon** [damianavila/RISE#585]
Hi, I can set slideshow in cell toolbar, but I cannot find the Enter/Exit Live Reveal Slideshow button (neither Alt-r is working). Maybe I need some other package? 

I already tried all restarts, starting new notebook, etc

Config:
- Debian
- Python 3.9
- Rise 5.7. 1
- Jupyter-lab 3.0.7

**Comment by parmentelat** [comment]
Hi

you have the right version of rise and no other package is required

I would recommend that you look in the browser console and search for any error that could give us a hint as to what's going on

also, generally speaking, it is frequent for similar problems to be linked with other extensions, do you have any uncommon jupyter extensions enabled ?
you can list extensions with `jupyter nbextension list`

**Comment by lbugnon** [comment]
Thanks for the reply! I listed the nbextensions but its empty. Turns out that I didn't have enabled the extensions by default in my user. Solved using nbextensions_configurator, but i don't know if its the best approach

**Comment by damianavila** [comment]
@lbugnon do you remember how you installed it? This seems to be an issue with the "environments" you are working with...

**Comment by lbugnon** [comment]
@damianavila nop sorry, I tried a lot of things. I think the bottom line is that rise is not enabled after installing with pip, thus I had to use nbextension_configuration to enable it and then it worked

**Comment by damianavila** [comment]
OK, thanks for the quick reply!!

**Comment by rwenz3l** [comment]
```
pip3 install --upgrade jupyter
pip3 install --upgrade rise
pip3 install --upgrade jupyter_nbextensions_configurator
jupyter nbextensions_configurator enable --user
jupyter notebook
# browser opens and you can hit 'Enable' on the RISE extension in a new tab called Nbextensions
```

I think the docs need to be updated to instruct users to `enable` the extension. It's obviously no longer doing that by default.

**Comment by parmentelat** [comment]
@rwenz3l 

can you please elaborate a little ? what are these commands meant to describe ? 
it is a scenario that you tried ? in that case why the upgrade of jupyter, does that matter ?

in my own use cases, a simple pip install does the trick for me as far as I can tell
so  I guess we're going to need a little more context to properly understand what the issue actually is in your setup

**Comment by damianavila** [comment]
Enable is no longer necessary since a long time ago in RISE ... not sure about the configurator, but I would expect the same behavior. As @parmentelat suggested, more info about your setup would be useful.

**Comment by rwenz3l** [comment]
Sorry for never expanding on this - I think the setup was a simple xubuntu 20.04 install and I installed jupyter with python3 and pip3. IIRC the RISE extension was not loaded properly. I can't recreate the scenario now.

> what are these commands meant to describe ?

I think after I purged all the jupyter stuff, these were the commands I used to get it finally going properly.
So they are meant to describe my solution :)

> why the upgrade of jupyter, does that matter ?

I don't think it matters, it just ensures you have the latest version.

I mainly wanted to throw it out here for anyone else that encountered an issue with the default install. I assume something in my env just wasn't right.
